perun-registrar-lib/src/test/java/cz/metacentrum/perun/registrar/ExpirationNotifSchedulerTest.java

```diff
@@ -686,4 +757,62 @@ private AttributeDefinition setUpGroupMembershipExpirationAttribute() throws Exc
 
 		return perun.getAttributesManager().createAttribute(session, attr);
 	}
+
+	/**
+	 * Converts object ApplicationFormItemWithPrefillValue to ApplicationFormItemData
+	 *
+	 * @param object ApplicationFormItemWithPrefilledValue to convert
+	 * @return converted object ApplicationFormItemData
+	 */
+	private ApplicationFormItemData convertAppFormItemWithPrefValToAppFormItemData(ApplicationFormItemWithPrefilledValue object) {
+		return new ApplicationFormItemData(object.getFormItem(), object.getFormItem().getShortname(), "", "0");
+	}
+
+	/**
+	 *
```

Comment:
You are missing here a comment about what the method does.

Reply:
Comment has been added